.github/actions/lint/action.yml

```diff
@@ -0,0 +1,13 @@
+runs:
+  using: composite
+  steps:
+    # Github's ubuntu-20.04 image already has clang-format-11 installed
+    - run: |
+        find src unittests -type f \( -name '*.cpp' -o -name '*.h' -o -name '*.ipp' \) -print0 | xargs -0 clang-format-11 -i
```

Comment:
Maybe we should use `clang-format` alias instead, WDYT? I doubt anyone would have anything older than `v11` either way and most will have `clang-format-14` on a modern system.